lms/templates/shoppingcart/error.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
 <%inherit file="../main.html" />
 <%!
 from django.utils.translation import ugettext as _
```

Comment:
if `${error_html}` below is really HTML, then this needs to be `${HTML(error_html)}`.  This all requires testing before further review.  Thanks.